docs/modules/ROOT/pages/references/glossary.adoc

```diff
@@ -15,18 +15,40 @@ It's the entity that owns the associated resources and the business connection.
 Billing is done on this level.
 
 Team::
-A group of users. 
+A group of users.
 Always belongs to exactly one organization.
 
 Role::
-A collection of permissions a subject has within an object (subjects would be User and Team, objects would be Organization, Team and Namespace) 
+A collection of permissions a subject has within an object (subjects would be User and Team, objects would be Organization, Team and Namespace)
 
 Namespace::
 A Kubernetes namespace (in OpenShift also an object type "Project")
 
 Zone::
 A single OpenShift cluster that is part of an {product} offering.
 
+Control API::
+An API endpoint that exposes certain {product} features to users.
+Specifically, this consists of:
+* API that users write to with various clients (for example frontend or CLI) and thus configure their organizations
+* API that users read with various clients that is usually the result of {product} components (for example read-only access to invoices)
+
+Plugin API::
+A subset of Control API that is specifically targeted for Plugins.
+This specification may not only be technical, but may also contain a "behavioral contract" that is in documentation only.
+It basically defines how Plugins should behave and interact with {product}.
```

Comment:
I'm not quite sure about the `a "behavioral contract" that is in documentation only`. With this sentence, I want to point out to the metering system idea: The plugin API in this case defines the contract how the metrics database is to be accessed, e.g. database and data scheme. In that case I consider the data scheme from $database to be the "API" that the plugin has to understand in order to make sense of the data.

I would be open to exclude this type of "contract" from this term and/or define another term for that though.